### PR TITLE
Fix php8.1 deprecations

### PR DESCRIPTION
**Pull request by bastien-phi**
Fixes deprecations for php8.1

### CODE REVIEW

File: src/Manipulators/Brightness.php [hunk 1]
```diff
@@ -34,6 +34,10 @@ public function run(Image $image)
      */
     public function getBrightness()
     {
+        if (null === $this->bri) {
```
Comment (ADmad) [inline src/Manipulators/Brightness.php:37]: A separate `if` block isn't necessary. You can just add `null === $this->bri || ` to the `if` block below.

**Comment by ADmad** [comment]
Please rebase from upstream master where I have fixed the testsuite failure on PHP 8.1.

**Comment by bastien-phi** [comment]
Should be all good !

File: src/Manipulators/Gamma.php [hunk 2]
```diff
@@ -34,6 +34,10 @@ public function run(Image $image)
      */
     public function getGamma()
     {
+        if (null === $this->gam) {
+            return;
+        }
+
         if (!preg_match('/^[0-9]\.*[0-9]*$/', $this->gam)) {
```
Comment (ADmad) [inline src/Manipulators/Gamma.php:41]: ... and here.

File: src/Manipulators/Contrast.php [hunk 3]
```diff
@@ -34,6 +34,10 @@ public function run(Image $image)
      */
     public function getContrast()
     {
+        if (null === $this->con) {
+            return;
+        }
+
         if (!preg_match('/^-*[0-9]+$/', $this->con)) {
```
Comment (ADmad) [inline src/Manipulators/Contrast.php:41]: Same change as the one I suggested for `Brightness` here too :slightly_smiling_face:.

**Comment by ADmad** [comment]
No problem, thanks.

**Comment by bastien-phi** [comment]
Glad to help !

**Pull request merged**